src/Yasumi/data/translations/goodFriday.php

```diff
@@ -26,8 +26,9 @@
     'en_ZA' => 'Good Friday',
     'es_ES' => 'Viernes Santo',
     'fi_FI' => 'Pitkäperjantai',
+    'fr_BE' => 'Vendredi Saint',
     'fr_CH' => 'Vendredi Saint',
-    'fr_FR' => 'Vendredi saint',
+    'fr_FR' => 'Vendredi Saint',
```

Comment:
This change cause the unit test for France to fail. Please correct the corresponding unit test for Good Friday (France) as well.

Reply:
Done